doc/active_docs/CMS API.json

```
@@ -262,11 +262,6 @@
               "paramType": "query",
               "allowMultiple": false
             },
-            {
-              "name": "system_name",
-              "description": "Human readable and unique identifier",
-              "paramType": "query"
-            },
             {
               "name": "title",
```

Do we need this as well?
```suggestion
              "name": "title",
              "required": true,
```

---

You are right